### ISSUE TITLE
i cant get any of your sources to work 

### ISSUE THREAD

**Issue by yugioh9** [Tempest0580/tempest#99]
iam haveing trouble with the  provders none of the are working at all  can you please fix it

**Comment by DJSleeves** [comment]
I think I'm having the same problem.  Does it keeps saying "no stream available"?

**Comment by yugioh9** [comment]
tes mine is saying that  all the provider come up but there no links

**Comment by yugioh9** [comment]
Yes it is saying the same thing man  on mine too  Sent from Mail for Windows 10 From: DJSleevesSent: June 16, 2021 11:27 PMTo: Tempest0580/tempestCc: yugioh9; AuthorSubject: Re: [Tempest0580/tempest] i cant get any of your sources to work (#99) I think I'm having the same problem. Does it keeps saying "no stream available"?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
